KMMYCharts/src/commonMain/kotlin/co/yml/kmm/charts/common/components/accessibility/SliceInfo.kt

```diff
@@ -0,0 +1,42 @@
+package co.yml.kmm.charts.common.components.accessibility
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.*
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.semantics.semantics
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import co.yml.kmm.charts.ui.piechart.models.PieChartData
+
+/**
+ * Composable to display each slice for a given pie chart.
+ * @param slice: Details of each slice in pie/ donut chart.
+ * @param slicePercentage: Percentage of each slice.
+ */
+@Composable
+internal fun SliceInfo(slice: PieChartData.Slice, slicePercentage: Int) {
+    // Merge elements below for accessibility purposes
+    Row(modifier = Modifier
+        .clickable { }
```

Comment:
Why do we need this method ?

Reply:
This is the method to show the pie chart slice when we click on slice info.